gems/aws-sdk-core/spec/aws/log/formatter_spec.rb

```diff
@@ -34,6 +34,10 @@ def format(pattern, options = {})
         end
 
         it 'provides a :request_params replacement' do
```

Comment:
If this test is acting like a kitchen sink, it would be appreciated if we can test all the possibilities in the case statement. Would you mind adding an array here too?

Reply:
done